CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 0.53.1 (2025-10-02)
+
+
+### Features
+* **experimental**: Add mcp to deadline client tools (#784) ([`af4064b`](https://github.com/aws-deadline/deadline-cloud/commit/af4064b1c0bb7455affa9987e6d031ac1e1f5afe))
+* Add '-p' param support to 'deadline bundle gui-submit' ([`53c507e`](https://github.com/aws-deadline/deadline-cloud/commit/53c507ed817ab9ffde501121515a83677d61db22))
+* Add debug snapshot option to 'deadline bundle submit' ([`c4c96b9`](https://github.com/aws-deadline/deadline-cloud/commit/c4c96b9235fceb96224715a7fc81afa66e2484b5))
+
+### Bug Fixes
+* concurrent job bundle submissions fail due to sqlite3 table already exist (#862) ([`b35aeb1`](https://github.com/aws-deadline/deadline-cloud/commit/b35aeb1eb3ae6955cd4f70f3c2c57f2b0d5bd900))
```

Comment:
For each of the bug fixes, we try to describe the bug being fixed, not the fix for the bug. I'd suggest going through each of these to modify the wording. For example, we don't need the details of adding a thread local s3 cache db, or multithreading syscalls, we can just scope it down to something like "Performance improvement when submitting a job'. or similar.  


There are some some bug fixes here which are actually for integ tests which should be excluded from the changelog